BALSAMIC/commands/config/case.py

```diff
@@ -188,6 +194,8 @@ def case_config(
         directory.mkdir(exist_ok=True)
 
     config_collection_dict = ConfigModel(
+        sentieon_install_dir=sentieon_install_dir if sentieon_install_dir else None,
+        sentieon_license=sentieon_license if sentieon_license else None,
```

Comment:
Here as well, it's a mandatory field for balsamic right now. We've been ignoring the initial idea of granularity with the `--disable-variant-callers`

```suggestion
        sentieon_install_dir=sentieon_install_dir,
        sentieon_license=sentieon_license,
```